tsconfig.json

```diff
@@ -5,11 +5,13 @@
     "removeComments": false,
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "moduleResolution": "Node",
+    "moduleResolution": "node",
     "module": "commonjs",
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowJs": true,
-    "typeRoots": ["../node_modules/@types", "./types/modules"]
-  }
+    "skipLibCheck": true
+  },
+  "include": ["./src/**/*.ts"],
```

Comment:
I think this file is only used as a base for the other tsconfig files, right? Why does it need to specify include? Shouldn't that be in the other files? Isn't that defined by webpack? It feels redundant or unused.

Reply:
Added a comment.